internal/lsp/server.go

```diff
@@ -461,10 +473,31 @@ func (l *LanguageServer) handleInitialize(
 		},
 	}
 
+	if l.clientRootURI != "" {
+		l.log("running in workspace mode")
+		l.workspaceMode = true
+
+		err = l.loadWorkspaceContents()
+		if err != nil {
+			return nil, fmt.Errorf("failed to load workspace contents: %w", err)
+		}
+
+		// attempt to load the config as it is found on disk
+		file, err := config.FindConfig(strings.TrimPrefix(l.clientRootURI, "file://"))
```

Comment:
Was file:// not an issue on windows? Or it’s not applicable here?

Reply:
Hmm, need to double check this. Will save a TODO for this.